app/src/main/java/com/paypal/psix/models/Event.java

```diff
@@ -4,11 +4,16 @@
 import com.activeandroid.annotation.Column;
 import com.activeandroid.annotation.Table;
 
+import org.parceler.Parcel;
+
 import java.util.List;
 
+@Parcel(value = Parcel.Serialization.BEAN, analysisLimit = Model.class)
```

Comment:
`analysisLimit` means that `Parceler` stops analyzing the model when it comes to the `ActiveAndroid`'s `Model` class (otherwise it crashes). They actually added this a month ago and it exists only in 0.2.16 version which was not released yet :)